man/man8/zed.8.in

```diff
@@ -231,12 +231,6 @@ Terminate the daemon.
 
 .SH BUGS
 .PP
-The ownership and permissions of the \fIenabled-zedlets\fR directory (along
-with all parent directories) are not checked.  If any of these directories
-are improperly owned or permissioned, an unprivileged user could insert a
-ZEDLET to be executed as root.  The requirement that ZEDLETs be owned by
-root mitigates this to some extent.
```

Comment:
This is still relevant and explains why the ZED requires that the scripts be owned by root.  Specifically for the reason you mentioned in your comment.  Even if the directories were incorrectly permissioned an unprivileged user couldn't add a new script which would be run as root unless they could also permission it as root.  So I think it'd be OK to drop the FIXME but this note should probably remain in the man page somewhere as an explanation for this requirement.

Reply:
I thought that was already mentioned up-page, but it may well not be, and I was just imagining it after staring at this for too long. Will add the zedlet ownership requirements.